src/applications/mhv/medications/containers/PrescriptionDetails.jsx

```diff
@@ -330,7 +338,9 @@ const PrescriptionDetails = () => {
       setPdfTxtGenerateStatus({ status: PDF_TXT_GENERATE_STATUS.NotStarted });
       setTimeout(() => window.print(), 1);
     }
-    dispatch(clearAllergiesError());
+    setTimeout(() => {
```

Comment:
might be good to add an if statement here so we can avoid having async updates whenever possible. Check if the has an error and clear only if it does.